airflow/Dockerfile

```diff
@@ -48,7 +48,10 @@ RUN python${PYTHON} -m venv --system-site-packages /stackable/app && \
     pip install --no-cache-dir apache-airflow[${AIRFLOW_EXTRAS}]==${PRODUCT} --constraint /tmp/constraints.txt && \
     # Needed for pandas S3 integration to e.g. write and read csv and parquet files to/from S3
     pip install --no-cache-dir s3fs cyclonedx-bom && \
-    cyclonedx-py environment --schema-version 1.5 --outfile /stackable/airflow-${PRODUCT}.cdx.json
+    cyclonedx-py environment --schema-version 1.5 --outfile /stackable/airflow-${PRODUCT}.cdx.json && \
+    # Needed for OIDC
+    pip install --no-cache-dir --upgrade Flask_OIDC==2.2.0 Flask-OpenID==1.3.1
```

Comment:
I'd assume that this needs to go before the cyclonedx command otherwise it probably won't be added to the SBOM

@dervoeti ?

Reply:
Yes that's correct. I'll add a comment, reworking the Airflow SBOM anyway at the moment.

Reply:
Excellent. Done with [8a2730c](https://github.com/stackabletech/docker-images/pull/891/commits/8a2730c58d1981aa75a0f723c10fb3487176bd7a)